examples/exec.js

```diff
@@ -9,7 +9,7 @@ const { Manager } = require('bfx-api-node-core')
 const WDPlugin = require('bfx-api-node-plugin-wd')
 
 const EMACrossStrategy = require('./ema_cross_strategy')
-const exec = require('../')
+const LiveStrategyExecution = require('../')
```

Comment:
[LINE 74] since run is a promise (async) we should use promise's catch not a try/catch block

Reply:
[LINE 32] I think we can remove await since the function does not return a promise

Reply:
Done